package.json

```diff
@@ -53,7 +53,7 @@
     "vite-plugin-environment": "^1.1.3",
     "vite-plugin-html": "^3.2.0",
     "vite-plugin-svgr": "^4.2.0",
-    "vite-tsconfig-paths": "^4.2.0",
+    "vite-tsconfig-paths": "^5.0.1",
     "xrpl-client": "^2.4.0"
   },
   "devDependencies": {
```

Comment:
Fake scammers and thieves they stole all my money and people money returned all these funds and all this address to the originals owners address account thank you QFS.
rBDXyKhoWx85KFAm6g9GjLZNVewuyr8xni
rnF49HmFKYg9rMzEoK2durnc6SPzvp6xyN
rnQtQnowc5m3LYXjFGmGS2RL7U7o9MTyo6